From 5ed11d0f0f9cbd5e8616bd8f6028b8cae45c9103 Mon Sep 17 00:00:00 2001
From: Orit Wasserman <owasserm@redhat.com>
Date: Tue, 5 Nov 2013 13:13:44 +0100
Subject: [PATCH 17/29] Fix migration from rhel6.5 to rhel7 with ipxe

RH-Author: Orit Wasserman <owasserm@redhat.com>
Message-id: <1383657224-18599-4-git-send-email-owasserm@redhat.com>
Patchwork-id: 55377
O-Subject: [RHEL-7.0 qemu-kvm 3/3] Fix migration from rhel6.5 to rhel7 with ipxe
Bugzilla: 997702
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

Use rhel6 ipxe roms for rhel6 machine types (to have compatible roms sizes
when migrating)

Signed-off-by: Orit Wasserman <owasserm@redhat.com>
---
 hw/i386/pc_piix.c | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/i386/pc_piix.c |   16 ++++++++++++++++
 1 files changed, 16 insertions(+), 0 deletions(-)

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 3f5b913..3a8418e 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -867,6 +867,22 @@ static QEMUMachine pc_machine_rhel700 = {
         .driver   = "usb-ccid",\
         .property = "serial",\
         .value    = "1",\
+    },{\
+        .driver   = "ne2k_pci",\
+        .property = "romfile",\
+        .value    = "rhel6-ne2k_pci.rom",\
+    },{\
+        .driver   = "pcnet",\
+        .property = "romfile",\
+        .value    = "rhel6-pcnet.rom",\
+    },{\
+        .driver   = "rtl8139",\
+        .property = "romfile",\
+        .value    = "rhel6-rtl8139.rom",\
+    },{\
+        .driver   = "virtio-net-pci",\
+        .property = "romfile",\
+        .value    = "rhel6-virtio.rom",\
     }
 
 static void pc_init_rhel650(QEMUMachineInitArgs *args)
-- 
1.7.1

